### DISCUSSION TITLE
I paid but now got a message that my trial period is over...

### DISCUSSION THREAD

**Discussion by MottiBazar** [community/community#48021] [New to GitHub] [answered]
Hello, I don't know who to contact about this issue and will appreciate your help with that.
A few months ago, I paid the yearly subscription fee and started using Copilot.
Today I received a message (I am using VSCode) that the trial period ended.
I will appreciate your help in telling me who to contact in order to enable my account.
Thanks!

**Comment by trwstin** [comment, marked as answer]
You can try opening a ticket for GitHub Support here: https://support.github.com/request

**Comment by MottiBazar** [comment]
Thanks a lot! Will try.
